packages/manager/modules/pci/src/projects/project/components/project-dashboard-activate-banner/index.scss

```diff
@@ -0,0 +1,3 @@
+.product-activate-banner {
+  top: -1.47rem !important;
+}
```

Comment:
Are you sure that is used ?

Reply:
This CSS class is used in the template packages/manager/modules/pci/src/projects/project/components/activate-project-banner/activate-project-banner.html
And is already loaded in packages/manager/modules/pci/src/projects/project/components/activate-project-banner/index.js